tests/IRITest.php

```diff
@@ -41,9 +41,16 @@
  */
 
 require_once dirname(__FILE__) . '/bootstrap.php';
- 
+
 class IRITest extends PHPUnit\Framework\TestCase
 {
+    public static function setUpBeforeClass()
+    {
+        if(class_exists('PHPUnit_Framework_Error_Notice')) {
+            class_alias('PHPUnit_Framework_Error_Notice', 'PHPUnit\Framework\Error\Notice');
+        }
+    }
+
```

Comment:
Again, I think we should just update PHPUnit to 6.x (which still supports PHP 5.6) and update all of the classname references.

Reply:
No, the PHPUnit ```6.x``` cannot support the PHP 5.6.
If we want to support the PHP 5.6, we have to use the PHPUnit 5 and 6.
See more details about the PHPUnit [doc](https://phpunit.de/).